apps/admin-server/src/pages/projects/[project]/comments/index.tsx

```diff
@@ -18,17 +18,35 @@ export default function ProjectComments() {
   const [comments, setComments] = useState<any[]>([])
 
   const exportData = (data: any[], fileName: string) => {
+    const flattenObject = (obj: any, parent: string = '', res: any = {}) => {
+      for (let key in obj) {
+        const propName = parent ? `${parent}.${key}` : key;
+        if (typeof obj[key] === 'object' && obj[key] !== null) {
+          if (Array.isArray(obj[key])) {
+            res[propName] = obj[key].map((item: any) => JSON.stringify(item)).join(',');
+          } else {
+            flattenObject(obj[key], propName, res);
+          }
+        } else {
+          res[propName] = obj[key];
+        }
+      }
+      return res;
+    };
```

Comment:
Graag deze functie lostrekken naar een extern bestand, zodat je die ook in resources/index.tsx en elders kunt hergebruiken